web-api/src/persistence/postgres/cases/getCaseByDocketNumber.ts

```diff
@@ -27,25 +27,28 @@ export const getCaseByDocketNumber = async ({
   // These case items are no longer in dynamoDB
   const SK_FILTER_OUT = ['work-item', 'correspondence', 'case'];
 
-  const dbCaseMetadata = await getCaseMetadataWithCounsel({
-    applicationContext,
+  const dbCaseMetadata = await getCaseMetadataByDocketNumber({
```

Comment:
We don't need to get the counsel--that will be gotten further down in the file as part of our catch-all dynamo query